### ISSUE TITLE
Does the jq mode recreate a child process for each message?

### ISSUE THREAD

**Issue by ozomer** [jimbojw/node-red-contrib-json#2]
This looks very inefficient.

**Comment by jimbojw** [comment]
Yes, it creates a process per message. I wanted to keep one process open and send a message at a time through, but couldn't get it to work at the time.

**Comment by ozomer** [comment]
Use the `--unbuffered` option. `--seq` is also useful to avoid crashing on error (each line should start with '\x1e' RS ascii char).
For example:
```
var jq = require('child_process').spawn('jq', ['.x', '--compact-output', '--unbuffered', '--seq']);
jq.stdout.pipe(process.stdout);
jq.stdin.write('\x1e{"x":2}\n'); // output is printed immediately
```

**Comment by jimbojw** [comment]
Ah I see. Yes, unbuffered was probably the magic that I needed. At the time I couldn't figure out how to get it to give me anything in a streaming way.
